.github/workflows/tag-publish.yaml

```diff
@@ -116,6 +116,15 @@ jobs:
             "${{ steps.image-name.outputs.tagged }}" \
             "${{ steps.image-name.outputs.name }}:latest"
 
+      # if: ${{ github.event_name != 'pull_request' }}
```

Comment:
Should this be added back so that we only publish on commits to dev?

Reply:
Yes, I will before putting it up for review. Just wanted to test out all the cases first.